capstone/src/test/java/com/google/sps/AuthServletTest.java

```diff
@@ -0,0 +1,70 @@
+package com.google.sps.servlets;
```

Comment:
nit: test directory should mirror the code directory. So there should also be .../servlets/auth/AuthServletTest.java, etc. Same for StudentServlet

Reply:
Noted, thanks for pointing this out :)